packages/docusaurus-theme-classic/src/theme/BlogTagsListPage/index.js

```diff
@@ -48,7 +48,7 @@ function BlogTagsListPage(props) {
     .filter(item => item != null);
 
   return (
-    <Layout title="Blog Tags" description="Blog Tags">
```

Comment:
What's wrong with Blog Tags here?

Reply:
I wanted to make the headline "more general", that it was suitable for both the regular using mode and the blog-only. This is all because on other blog pages it is difficult to determine if blog-only mode is enabled. I only had an idea with a global variable on my mind (via webpack), but I decided that I could reduce my efforts and slightly change the title :man_shrugging: 

Should I revert this change?